src/platform/web/Platform.js

```diff
@@ -307,6 +315,48 @@ export class Platform {
         return DEFINE_VERSION;
     }
 
+    get themes() {
+        return Object.keys(this.config["themes"]);
+    }
+
+    async getActiveTheme() {
+        // check if theme is set via settings
+        let theme = await this.settingsStorage.getString("theme");
+        if (theme) {
+            return theme;
+        }
+        // return default theme
+        if (window.matchMedia) {
+            if (window.matchMedia("(prefers-color-scheme: dark)")) {
+                return this.config["defaultTheme"].dark;
+            } else if (window.matchMedia("(prefers-color-scheme: light)")) {
+                return this.config["defaultTheme"].light;
+            }
+        }
+        return undefined;
+    }
+
+    setTheme(themeName) {
+        const themeLocation = this.config["themes"][themeName];
+        if (!themeLocation) {
+            throw new Error(`Cannot find theme location for theme "${themeName}"!`);
+        }
+        this._replaceStylesheet(themeLocation);
```

Comment:
as mentioned in the standup, to make the config file forwards-compatible, we should perhaps already load the css file location from the theme manifests rather than have the css location straight in the config file. As you said, we'll need this anyway for runtime themes.